src/perf/bin/drvmain.cpp

```diff
@@ -0,0 +1,604 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    QUIC Kernel Mode Performance Driver
+
+--*/
+
+#include <quic_platform.h>
+#include "msquic.h"
+#include <quic_driver_main.h>
+#include "perfioctls.h"
```

Comment:
let's put a _ between the words of this file name: "perf_ioctls.h"

Reply:
This was renamed to use capitals to match PerfHelpers and other things.